backends/dpdk/dpdkHelpers.cpp

```diff
@@ -346,6 +437,166 @@ bool ConvertStatementToDpdk::preorder(const IR::AssignmentStatement *a) {
     return false;
 }
 
+/* This function processes the hash parameters and stores it in
+   "components" which is further used for generating hash instruction
+*/
+void ConvertStatementToDpdk::processHashParams(const IR::Argument* field,
+                                               IR::Vector<IR::Expression>& components) {
+    if (auto params = field->expression->to<IR::Member>()) {
```

Comment:
"params" suggests a parameter. Do you want to call it something else?
(it's not an IR::Parameter)

Reply:
Also, perhaps you want a singular name, e.g. exp.

Reply:
Updated